FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Metrics/ProcessLatencyMetric.cs

```diff
@@ -10,9 +10,9 @@ namespace Microsoft.Health.Fhir.Synapse.Common.Metrics
     public class ProcessLatencyMetric : DiagnosticMetrics
     {
         public ProcessLatencyMetric()
-            : base("ResourceLatency", MetricsCategories.Latency, new Dictionary<string, object>
+            : base("ProcessLatency", MetricsCategories.Latency, new Dictionary<string, object>
             {
-                { DimensionNames.Name, "ResourceLatency" },
+                { DimensionNames.Name, "ProcessLatency" },
```

Comment:
Will this name change impact PaaS metrics? Not sure if this is used as the metrics key